WordPressEditor/WordPressEditor/Classes/Plugins/WordPressPlugin/Gutenberg/GutenbergOutputHTMLTreeProcessor.swift

```diff
@@ -129,21 +129,24 @@ private extension GutenbergOutputHTMLTreeProcessor {
     // MARK: - Gutenberg HTML Attribute Data
     
     private func gutenblockCloserData(for element: ElementNode) -> String? {
-        return decodedAttribute(named: GutenbergAttributeNames.blockCloser, from: element)
+        let decoder = GutenbergAttributeDecoder()
+        return decoder.decodedAttribute(named: GutenbergAttributeNames.blockCloser, from: element)
     }
     
     private func gutenblockOpenerData(for element: ElementNode) -> String? {
-        return decodedAttribute(named: GutenbergAttributeNames.blockOpener, from: element)
+        let decoder = GutenbergAttributeDecoder()
+        return decoder.decodedAttribute(named: GutenbergAttributeNames.blockOpener, from: element)
     }
     
     private func gutenblockSelfCloserData(for element: ElementNode) -> String? {
-        return decodedAttribute(named: GutenbergAttributeNames.selfCloser, from: element)
+        let decoder = GutenbergAttributeDecoder()
+        return decoder.decodedAttribute(named: GutenbergAttributeNames.selfCloser, from: element)
     }
 }
 
 // MARK: - HTML Attributes
 
-private extension GutenbergOutputHTMLTreeProcessor {
+public class GutenbergAttributeDecoder {
```

Comment:
It was a very good call to turn this into a separate class, but I'd also move this to a separate file since it's no longer part of `GutenbergOutputHTMLTreeProcessor` (especially since it's public).